openslides/users/config_variables.py

```diff
@@ -11,7 +11,7 @@ def get_config_variables():
     # Sorting
     yield ConfigVariable(
         name='users_sort_by',
-        default_value='first_name',
+        default_value='firstName',
```

Comment:
Yes, but now we use this config variable on client side only. There we use CamelCase. So I don't want to change it again back.